osutil/find_in_path.go

```diff
@@ -38,13 +36,9 @@ func FindInPath(name string) string {
 // listed in the environment variable PATH and returns the found path or the
 // provided default path.
 func FindInPathOrDefault(name string, defaultPath string) string {
```

Comment:
Sorry for being complicated, I wonder if maybe `LookupPathWithDefault()` so that its closer to the exec.LookupPath (i.e. less cognitive load by having "FindInPath" vs "LookupPath" depending on what is needed).